cmd/brew-test-bot.rb

```diff
@@ -132,8 +132,8 @@ def resolve_test_tap
       end
     end
 
-    return unless git_url = ENV["UPSTREAM_GIT_URL"] || ENV["GIT_URL"]
-    # Also can get tap from Jenkins GIT_URL.
+    # Get tap from Jenkins GIT_URL or Circle CIRCLE_REPOSITORY_URL.
+    return unless git_url = ENV["UPSTREAM_GIT_URL"] || ENV["GIT_URL"] || ENV["CIRCLE_REPOSITORY_URL"]
```

Comment:
I changed it to
```ruby
    # Get tap from Jenkins UPSTREAM_GIT_URL, GIT_URL or
    # Circle CI's CIRCLE_REPOSITORY_URL.
    git_url =
      ENV["UPSTREAM_GIT_URL"] ||
      ENV["GIT_URL"] ||
      ENV["CIRCLE_REPOSITORY_URL"]
    return unless git_url
```
If you prefer your suggested style above, I'll change it toute-de-suite.